dashboard.csss-ci.com/server/api/controllers/rest/v1/Banks.php

```diff
@@ -85,7 +101,13 @@ public function index_put($id) {
             $this->response(['Bad Request'], REST_Controller::HTTP_BAD_REQUEST);
         }
     }
-
+    /**
+     * sets up banker user with id, bankid, banker name, phone, email, and password and saves it to banker_model query
+       also sets up html to display fields that banker user sees and needs to input 
```

Comment:
I don't completely understand what you mean by this, but I do not believe that the program does this